.bazelrc

```diff
@@ -36,6 +36,12 @@ build --sandbox_default_allow_network=false
 # leaking into the build.
 build --incompatible_strict_action_env
 
+# By default add nice debug information for seastar and randomize the task
+# queue. These all get disabled in the release build.
+build --@seastar//:shuffle_task_queue=True
+build --@seastar//:debug=True
```

Comment:
These are very slow, I don't know if you want this for the "default" dev build?

I think we need to define how which primary flavors of build we have, in order to see what can have what.

In cmake build we basically have:

release build (seastar allocator, -O2)
dev build (mostly similar to release but 2x faster build, -O1, seastar allocator, runtime about 0.75 of release)
debug build (libc allocator, -O1, sestar debug stuff like task shuffling, ASAN, UBSAN, much slower runtime, maybe 0.1 of release)

Feels like we probably want at least those 3 flavors still? Not sure what the "default" should be, i.e., if you don't pass --config at all, maybe debug? Main downside of debug is how slow it is.

Reply:
I'm not ready to peel this onion yet - so I will remove and punt to another change.

Reply:
yeah makes sense to just do the "guard" thing here